examples/framework-google-functions-framework/README.md

```diff
@@ -0,0 +1,38 @@
+# Inngest Google Functions Framework Template
+
+This is a [Google Functions Framework](https://github.com/GoogleCloudPlatform/functions-framework-nodejs) project. It is a reference on how to send and receive events with Inngest and Google's Functions Framework for Node.js. Functions Framework is an Express.js compatible API.
+
+## Getting Started
+
+Use [`create-next-app`](https://www.npmjs.com/package/create-next-app) with [npm](https://docs.npmjs.com/cli/init), [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/), or [pnpm](https://pnpm.io) to bootstrap the example:
+
+```bash
+npx create-next-app --example https://github.com/inngest/inngest-js/tree/main/examples/framework-google-functions-framework inngest-google-functions
+```
+
+```bash
+yarn create next-app --example https://github.com/inngest/inngest-js/tree/main/examples/framework-google-functions-framework inngest-google-functions
+```
+
+```bash
+pnpm create next-app --example https://github.com/inngest/inngest-js/tree/main/examples/framework-google-functions-framework inngest-google-functions
+```
+
+Enter the directory and run:
+
+```bash
+# Transpile TypeScript to JavaScript
+npm run build
+# Run the service
+npm start
+```
```

Comment:
@jpwilliams - I noticed that other READMEs don't have a command on how to run the service itself. Is that something that we should add to all READMEs or does the create-next-app give more info on that? This has a discrete build step, so I thought it was worth calling this out. What do you think?

Reply:
Yeah, that'd be good! `create-next-app` does indeed tell you what to do (`cd` in and `npm run dev`), but it's still a missing step on the readme.